src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/merchandise_return.yml

```
@@ -23,3 +23,27 @@ admin_merchandise_returns_filter:
     redirectRoute: admin_merchandise_returns_index
     _legacy_controller: AdminReturn
 #    _legacy_link: AdminReturn:submitFilterorder_return
+
+admin_order_returns_edit:
+    path: /{orderReturnId}/edit
+    methods: [GET]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Sell/CustomerService/MerchandiseReturn:edit
```

```suggestion
        _controller: PrestaShopBundle\Controller\Admin\Sell\CustomerService\MerchandiseReturnController::editAction
```

We must use full notation now since this PR has been merged https://github.com/PrestaShop/PrestaShop/pull/25052, for consistency but mostly because short notation will no longer be supported in the future